clients/hadoopfs/src/main/java/io/lakefs/auth/GeneratePresignGetCallerIdentityRequest.java

```diff
@@ -0,0 +1,38 @@
+package io.lakefs.auth;
```

Comment:
Nit: this is a value type.  Could we use [org.immutables.value](https://immutables.github.io/)?